README.md

```diff
@@ -711,6 +711,17 @@ const view = instance.views.getView('zoomview');
 view.setAmplitudeScale(1.0);
 ```
 
+## Cue events
+
+Emit events when the playhead reaches a point or segment boundary:
+
+```js
+const cues = new Peaks.CueEmitter(instance);
```

Comment:
I would prefer to extend the existing API, rather than introduce a new class

```javascript
peaks.on('segments.enter', function(segment) { ... });
peaks.on('segments.exit', function(segment) { ... });
peaks.on('points.enter', function(point) { ... });
```